mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -505,10 +505,13 @@ private boolean userHasLookupRightsOn(Mailbox mailbox, MailboxSession session) t
                     if (!mailboxExists(mailbox, mailboxSession)) {
                         Mailbox m = doCreateMailbox(mailbox, mailboxSession);
                         MailboxMapper mapper = mailboxSessionMapperFactory.getMailboxMapper(mailboxSession);
-                        mapper.execute(Mapper.toTransaction(() -> mailboxIds.add(mapper.save(m))));
-
-                        // notify listeners
-                        dispatcher.mailboxAdded(mailboxSession, m);
+                        try {
+                            mapper.execute(Mapper.toTransaction(() -> mailboxIds.add(mapper.save(m))));
+                            // notify listeners
+                            dispatcher.mailboxAdded(mailboxSession, m);
+                        } catch (MailboxExistsException e) {
+                            // ignore
```

Comment:
Please can you add a log.

Reply:
Needs to be mentionned here: https://github.com/linagora/james-project/pull/1545

Ps: I'll drop these commits: no more required here.